sample.config.toml

```diff
@@ -9,6 +9,10 @@
 [templates]
   default = ''
 
+[remotes]
+  nfs = { name = '', url = '' }
+  smb = { name = '', url = '' }
```

Comment:
Do I have to put it as pre-created resources?

Reply:
These resources should be created on each test or one time for all tests?

Reply:
I put it here, because we have only one test with special remotes. What do you think ?

Reply:
Yes, but we need also to think about the resources usability.
If n tests want to create files on a NFS server, should they need n remotes for that or they can use the same remote?

Reply:
I think I should put the nfs config in the `preCreatedResources`.